assistant/src/test/java/com/ibm/watson/assistant/v1/model/UpdateDialogNodeTest.java

```diff
@@ -0,0 +1,366 @@
+/*
```

Comment:
Same question with this file? Did it not exist before

Reply:
it's a unit test generated by the merge-patch